scripts/g.extension/g.extension.py

```diff
@@ -388,16 +398,31 @@ def etree_fromfile(filename):
 
 def etree_fromurl(url):
     """Create XML element tree from a given URL"""
+    base_message = "Download file from <{url}> failed".format(url=url)
     try:
         file_ = urlopen(url)
+    except HTTPError as err:
+        if err.code == 404:
+            gscript.fatal(
+                _(
+                    "{base_message}. File not found on the"
+                    " server.".format(base_message=base_message)
+                ),
+            )
+        else:
+            gscript.fatal(
+                _(
+                    "{base_message}, {err}. ".format(
```

Comment:
Why comma here? Probably, neither `,` nor `.` are needed here. It should be `"{base_message}: {err}"`.